src/main/kotlin/com/vacgom/backend/global/config/LoggerConfig.kt

```diff
@@ -0,0 +1,15 @@
+package com.vacgom.backend.global.config
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class LoggerConfig {
+
+    @Bean
+    fun logger(): Logger {
+        return LoggerFactory.getLogger(this.javaClass)!!
```

Comment:
```suggestion
        return LoggerFactory.getLogger(this.javaClass)
```

타입을 보니 !! operator는 없어도 될것 .. 같습니다 !

Reply:
수정완료! 예리한 리뷰 감사드립니다 😄